### ISSUE TITLE
How does this license handle redistribution and modifications?

### ISSUE THREAD

**Issue by Amolith** [berneout/big-time-public-license#25]
As the title says, I don't see anything in [the license text](https://bigtimelicense.com/versions/2.0.1) that specifically addresses redistribution and modification rights. Is it designed to *accompany* something like the 3-Clause BSD License or would those rights fall under the Copyright and Patent license sections?

In either case, I think explicitly addressing those rights in the license text, whether by indicating that the reader should refer to an additional, included license, by laying them out inline, or by indicating that they fall under the Copyright/Patent License sections, would improve the Big Time Public License's clarity.

**Comment by kemitchell** [comment]
@Amolith the license is complete on its own. It's only the possible existence of exclusive copyrights or patent rights that make a license for the software necessary in the first place.

**Comment by lsmag** [comment]
Glad I found this question. I sketched somewhere in my notes to research later whether it made sense to combine BSD3+BTL or Parity+BTL. Imagining a Parity+BSL scenario, does this make sense?

- if you're using the software in order to build or work on open source, you fall under the Parity terms
- if you are not working on open source but it is for personal purposes, then Big time's terms apply
- only for companies above a certain specified threshold should seek a license

I also understand it's not something that needs to be addressed by Big Time or any other license, but another concern of mine was contributor code. Say, if contributors license their contributions with Parity, can I use it freely in my project using Parity+BTL?

**Comment by kemitchell** [comment]
@lsmag to be completely honest: I strongly suspect, based on experience over the last few years, that complex combinations of licenses are an antipattern. They become essentially un-grokable for even well meaning and interested potential users.

I'd strongly recommend composing a single statement of your public terms for free-of-charge use.

**Comment by lsmag** [comment]
> I strongly suspect, based on experience over the last few years, that complex combinations of licenses are an antipattern

Makes sense to me, I will stick with Big Time License then. It doesn't cover, however, a case that the likes of Business Source License and the Commons Clause cover. Is your opinion still the same with combining, say, Commons Clause with Big Time License?

**Comment by kemitchell** [comment]
Business Source and Commons Clause aren't so much licenses as license mix-ins or add-ons. They don't stand alone as licenses.

Even so, even the founding proponents of Commons Clause gave it up for purpose-drafted licenses.
